source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,56 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+Open a Change Stream
+--------------------
+
+You can keep track of changes of data in MongoDB, such as changes to a
+collection, database, or deployment by opening a :manual:`Change
+Stream</changeStreams/>`. A ``Change Stream`` allows applications to
+watch for changes to data and react to those changes. You can open a
+``Change Stream`` by calling the :node-api:`watch()
+</Collection.html#watch>` method on a collection, database, or
+deployment. 
+
+The ``watch()`` method takes an optional pipeline as the first parameter
+that is an array of stages to filter the :manual:`change events
+</reference/change-events/>` output. Also, ``watch()`` can take in
+additional options object as the second parameter. Set ``fullDocument``
+field of the additional options object to ``'updateLookup'`` to receive
+an event that contains the changes to the document as well as the entire
+altered document. Alternatively, you can leave  ``fullDocument`` or set it to the string ``'default'`` to only receive the altered document, but not the changes to the document. 
```

Comment:
I think you may have an incomplete thought: "you can leave ``fullDocument`` or...". I'm not certain what it means to do that -- did you mean leave it blank? Also, could you fix the column wrap when you get the chance?

Reply:
Changed to - " Alternatively, you can set ``fullDocument`` to the
string ``'default'`` to only receive the altered document, but not the
changes to the document." 

and wrapped